app/views/tag/show/_tab_content.html.erb

```diff
@@ -46,7 +46,11 @@
 
   <% if @node_type == "maps" %>
     <div class="tab-pane active" id="wikis">
-      <%= render :partial => "map/maps" %>
+     <% if @maps.nil? %>
```

Comment:
```suggestion
     <% if @nodes.nil? %>
```

It should be @nodes. If you test the route for valid tag, it's still giving error.

Reply:
done!